src/test/ui/feature-gate/stability-attribute-consistency.rs

```diff
@@ -11,6 +11,12 @@
 #![stable(feature = "stable_test_feature", since = "1.0.0")]
 
 #![feature(staged_api)]
+// Right now, stability attributes are only checked for consistency if we're
+// collecting lib feature attributes, for performance. We collect lib
+// feature attributes if there are non-lang feature attributes, or a crate
+// that depends on the current one has non-lang feature attributes. Thus,
+// we're enabling an arbitrary lib feature to force the check to kick in.
+#![feature(rustc_private)]
```

Comment:
It seems a bit off that this is necessary? Isn't there a way of making sure these checks always run, if they are needed?

Reply:
I should be able to refactor this so we always get the errors.